src/database/mod.rs

```diff
@@ -55,6 +55,18 @@ pub struct SyncTime {
     pub block_time: BlockTime,
 }
 
+/// Structure encapsulates criteria used for deleting
+/// spent UTXOs
+#[derive(Clone, Debug)]
+pub struct DelCriteria {
+    /// Number of Confirmation on UTXOs to
+    /// make it eligible for deletion
+    pub confirmations: Option<u32>,
+    /// Max number of allowable spent
+    /// UTXOs in the database
+    pub threshold_size: Option<u64>,
+}
```

Comment:
The comments over the field could be made in a single line.. They are not that big..

Reply:
Okay, I will change this.